Cargo.toml

```diff
@@ -25,108 +14,76 @@ members = [
 sp-crypto-hashing = { default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sp-io = { default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 sp-inherents = { default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 sp-runtime = { default-features = false, features = ["serde"], git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 sp-runtime-interface = { default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sp-staking = { default-features = false, features = ["serde"], git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 sp-storage = { default-features = false, features = ["serde"], git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 sp-version = { default-features = false, features = ["serde"], git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sp-weights = { default-features = false, features = ["serde"], git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 
 # substrate std / wasm only
-frame-support = { optional = true, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+frame-support = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+frame-system = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-assets = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-contracts = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-identity = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-recovery = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-society = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+pallet-staking = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sc-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sp-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
 
 # local deps
-ac-compose-macros = { path = "compose-macros", version = "1.17", default-features = false }
-ac-node-api = { path = "node-api", version = "1.17", default-features = false }
-ac-primitives = { path = "primitives", version = "1.17", default-features = false }
+ac-keystore = { default-features = false, path = "keystore", version = "1.17" }
+ac-compose-macros = { default-features = false, path = "compose-macros", version = "1.17" }
+ac-node-api = { default-features = false, path = "node-api", version = "1.17" }
+ac-primitives = { default-features = false, path = "primitives", version = "1.17" }
+substrate-api-client = { default-features = false, path = "api-client", version = "1.17" }
```

Comment:
@Niederb regarding your https://github.com/scs/substrate-api-client/pull/848#discussion_r2073530501. Yes the features are necessary, as we disable the default features here when we define the api-client as a workspace dependency here.

We could also do it the other way around as we need the default-features except for the no-std tests, but I decided to be consistent with the rest.

Reply:
Oh, I see. Makes sense then 👍

Reply:
We could also explicitly put `default-features = true` in the async example to overwrite the workspace dependency setting. Not sure what is better though.